heron/storm/src/java/shade.conf

```diff
@@ -0,0 +1,4 @@
+rule com.google.** org.apache.storm.shaded.@0
+rule com.twitter.** org.apache.storm.shaded.@0
```

Comment:
In general I can see a point - `heron-storm` artifacts aims to provide storm api, and therefore heron inside is impl detail. But I guess it's better to expose dependency on `heron-api`, for different config related things for example.